api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,16 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="Concept" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
+      </comment>
+      <addColumn tableName="Concept">
+            <column name="category" type="java.lang.Boolean"/>
```

Comment:
Is the category field defined it the java class as type boolean?

Reply:
The category field is of type concept

Reply:
So how does that translate to boolean?

Reply:
Running concept as a type returns an error during the local build

Reply:
Did you take a look at existing openmrs hibernating files that have the type concept?

Reply:
@dkayiwa I don't seem to be able to notice a mapping file with type concept

Reply:
Can you point me to the folder that you are looking at?

Reply:
https://github.com/openmrs/openmrs-core/tree/master/api/src/main/resources/org/openmrs/api/db/hibernate

Reply:
Did you check the Drug.hbm.xml file?

Reply:
Didn't notice that file! let me check it out

Reply:
Hey @dkayiwa I think I still I don't get what type is meant in this context as far as the hibernate mapping file is concerned for the category field

Reply:
The type in the Java class should be the same as in the hibernate mapping file. Did you check the Drug.hbm.xml file for an example? What did the hibernate documents that you read say?

Reply:
the docs said the types are as in the class as well, but types were all listed as data types like doubles, strings and yet here they don't build. so am a-little confused

Reply:
Did you try to put, in the hibernate mapping file, the category field type as in the Java class? If yes, i do not see it reflected in your pull request.

Reply:
I didn't push those changes because of the local build fails, but let me then push

Reply:
Did you really take a look at Drug.hbm.xml?